spyder/plugins/ipythonconsole.py

```diff
@@ -1050,15 +1046,194 @@ def additional_options(self):
         in JupyterWidget config options
         """
         options = dict(
-            pylab = self.get_option('pylab'),
-            autoload_pylab = self.get_option('pylab/autoload'),
-            sympy = self.get_option('symbolic_math'),
-            light_color = self.get_option('light_color'),
-            show_banner = self.get_option('show_banner')
+            pylab=self.get_option('pylab'),
+            autoload_pylab=self.get_option('pylab/autoload'),
+            sympy=self.get_option('symbolic_math'),
+            show_banner=self.get_option('show_banner')
         )
 
         return options
 
+    def get_style_sheet(self):

```

Comment:
Let's move this method as a function in `utils/ipython/syntax_style.py`. I suggest this for two reasons:

1. We don't need any attribute or method of `IPythonConsole` to compute this, only the color scheme values saved in our config system.
1. It wouldn't add more code to this (already) big plugin.